### PR TITLE
remove ashish-amarnath from maintainers

### PR DESCRIPTION
**Pull request by ashish-amarnath**
Signed-off-by: Ashish Amarnath <ashisham@vmware.com>

### CODE REVIEW

**Review by dsu-igeek — APPROVED** [review]
Thanks!

File: osstp.yaml [hunk 1]
```diff
@@ -0,0 +1,233 @@
+other:velero-plugin-for-aws:v1.1.0:
```
Comment (zubron) [inline osstp.yaml:1]: This file should be removed from this PR.

**Review by zubron — CHANGES_REQUESTED** [review]
Thanks for everything, @ashish-amarnath! There is just an extra file in this PR that I don't think should have been included.

**Review by zubron — APPROVED** [review]
Thanks, @ashish-amarnath!

**Review by dsu-igeek — APPROVED** [review]
Thanks Ashish!

**Pull request merged**